<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDocumentDialog</class>
 <widget class="QDialog" name="NewDocumentDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Document</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/breezEd.qrc">
    <normaloff>:/breezEd/icons/file/new</normaloff>:/breezEd/icons/file/new</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="typeList">
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descLabel">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Select a document type...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="pathLayout">
       <item>
        <widget class="QLineEdit" name="pathEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pathBrowseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/breezEd.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>browseForPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>typeSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseForPath()</slot>
  <slot>typeSelectionChanged()</slot>
 </slots>
</ui>
